runners/jstorm/src/main/java/org/apache/beam/runners/jstorm/translation/ParDoBoundTranslator.java

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
I think we can remove this class because SingleOutput will be translated to MultiOutput by default.